fish_command_timer.fish

```diff
@@ -123,7 +123,7 @@ else
   set -e fish_command_timer_enabled
 end
 
-set fish_command_timer_start_time
+set fish_command_timer_start_time (fish_command_timer_get_ts)
```

Comment:
This shouldn't be necessary as the preexec function below (`fish_command_timer_preexec`) will set this variable before each command is executed.